.github/actions/release/changelog-generator/action.yml

```
@@ -45,6 +45,11 @@ runs:
           --future-release ${{ inputs.future-release }} 
           ${{ inputs.optional-arg }}
 
+    - name: Enrich changelog
```

Opinion: maybe `changelog-descriptions` might be a more descriptive name for the script than `enrich-changelog`, or `populate-changelog-descriptions`?

```suggestion
    - name: Populate changelog descriptions
```